cmd/containerd-nydus-grpc/main.go

```diff
@@ -94,7 +94,7 @@ func main() {
 				return errors.Wrap(err, "set up logger")
 			}
 
-			log.L.Infof("Start nydus-snapshotter. PID %d Version %s FsDriver %s DaemonMode %s",
+			log.L.Infof("Start nydus-snapshotter. PID: %d Version :%s FsDriver: %s DaemonMode: %s",
```

Comment:
Should we make the colon very close to "Version" like `Version: ` `PID: ` ?

Reply:
OK, I will fix it.